python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -323,17 +288,6 @@ def trackandRelease(self):
 
                  flush_thread = threading.Thread(target=self._flush, args=())
                  flush_thread.run()
-
-                 for obj in self.dirty_objs.itervalues():
-
-                    if self.shouldReleaseRun is False:
-                        break
-
-                    _args = ([obj])
-                    release_thread = threading.Thread(target=self._release_lock, args=_args)
```

Comment:
OK, I think the line ```self._releaseWriteAccess()``` is present at the end of the Prep registry methods ```increase``` and ```decrease``` so I'm happy to keep this code removed :)